### PR TITLE
refactor: read parameters from CLI and ENV

### PR DESCRIPTION
**Pull request by falox**

### CODE REVIEW

File: cmd/cmd.go [hunk 1]
```diff
@@ -17,15 +19,31 @@ var rootCmd = &cobra.Command{
...
-// Refresh interval in seconds.
-var interval int
-
 var serveCmd = &cobra.Command{
 	Use:   "serve",
 	Short: "Start the server",
 	Long:  "Start the server to expose the metrics for the health analyzer",
 	Run: func(cmd *cobra.Command, args []string) {
-		server.StartServer(time.Duration(float64(interval) * float64(time.Second)))
+		url := cmd.Flags().Lookup("url").DefValue
```
Comment (iNecas) [inline cmd/cmd.go:27]: I suggest `prom-url` to have some consistency between the ENV vars and the CLI ones.

**Comment by iNecas** [comment]
Two small comments inline.

File: cmd/cmd.go (hunk 1 above)
Comment (falox) [reply cmd/cmd.go:27]: Right, let's be consistent. What about:

| CLI | CLI shortcut  | ENV  |
|--|--|--|
| --refresh-interval | -i | REFRESH_INTERVAL |
| --prom-url | -u | PROM_URL |

Also, what about expanding to `prometheus-url`?

**Review by xiormeesh — APPROVED** [review]
nice work on moving prom_url out of the code and into cmd params, much cleaner now!

**Pull request merged**